ksqldb-serde/src/main/java/io/confluent/ksql/serde/connect/ConnectFormat.java

```diff
@@ -54,6 +54,9 @@
  */
 public abstract class ConnectFormat implements Format {
 
+  public static String SR_KEY_SCHEMA_ID = "SR_KEY_SCHEMA_ID";
```

Comment:
Why the `SR_` prefix? Just curious? Isn't a schema-id not alway from SR?

Reply:
Good point. I changed this to `KEY_SCHEMA_ID` in next PR. I'll change it in this PR